website/docs/r/rdns.html.md

```diff
@@ -26,6 +27,19 @@ resource "hcloud_rdns" "master" {
 }
 ```
 
+For Floating IPs:
+```hcl
+resource "hcloud_floating_ip" "floating1" {
+  home_location = "ngb1"
+  type = "ipv4"
+}
+
+resource "hcloud_rdns" "floating_master" {
+  server_id = "${hcloud_floating_ip.floating1.id}"
```

Comment:
i don't understand why `server_id` is `floating.id` here. shouldn't it map to a server id or shouldn't we use `floating_ip_id` here?

Reply:
Sorry this was just a typo. I have fixed it.